### DISCUSSION TITLE
How to maximize inline gallery?

### DISCUSSION THREAD

**Discussion by mumus** [sachinchoolur/lightGallery#1279] [General]
Hello. Please, help. How to maximize inline gallery by clicking on current image, but not by clicking showMaximizeIcon? Thank u.
![image](https://user-images.githubusercontent.com/3994560/160092368-698e66a6-8f46-41d4-b465-0b4a880c451f.png)

**Comment by sachinchoolur** [comment]
Hey @mumus,

There is no default method available to maximize the inline gallery. 
But, you can do it by using lightGallery events and methods 

Here is the demo - https://codepen.io/sachinchoolur/pen/vYpxRGO

```js

inlineGallery.outer.on("click", (e) => {
  const $item = inlineGallery.outer.find(".lg-current .lg-image");
  if (
    e.target.classList.contains("lg-image") ||
    $item.get().contains(e.target)
  ) {
    inlineGallery.$container.removeClass("lg-inline");
  }
});
```

**Reply by mumus** [reply]
Very good. Thank u. Maybe better to do some opiton as default method?
